addons/l10n_latam_invoice_document/security/ir.model.access.csv

```diff
@@ -1,4 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_l10n_latam_document_type,access_l10n_latam_document_type,model_l10n_latam_document_type,base.group_user,1,0,0,0
 access_l10n_latam_document_type_readonly,l10n_latam.document.type.readonly,model_l10n_latam_document_type,account.group_account_readonly,1,0,0,0
 access_l10n_latam_document_type_invoice,l10n_latam.document.type.invoice,model_l10n_latam_document_type,account.group_account_invoice,1,0,0,0
```

Comment:
Again, `group_account_readonly` and `group_account_invoice` have `group_user` implied, so I guess both can be removed since they have the same access rights